src/metalookup/core/metadata_manager.py

```diff
@@ -101,10 +101,26 @@ async def extract(self, message: Input, extra: bool) -> Output:
                     tld_extractor=self.tld_extractor,
                 )
             self.logger.info(f"Built WebsiteData object in {t():5.2f}s.")
+            # fixme: For now we simply assume that the last entry is the relevant one.
+            #        This may be fixed together with https://github.com/openeduhub/metalookup/issues/85
+            response_code = site.har.log.entries[-1].response.status
+            # fixme: Eventually it is better to check for the "acceptable cases" (e.g. 200) and respond with a
```

Comment:
@RMeissnerCC what are your thoughts on that? 

We could also go with `if response_code != 200` because every thing else would mean, that we are not extracting from the actual content that was requested.

Reply:
I would like this if response_code != 200: raise HTTPException. Because we do not care what kind of exception it is, we just want success (capitalism 101 :D )

Reply:
yeah, i think thats more reasonable. changed it accordingly. The other fixme needs a little bit more thought, hence also the reference to #85